virttest/libvirt_vm.py

```diff
@@ -378,6 +379,15 @@ def backup_xml(self, active=False):
             LOG.error("Failed to backup xml file:\n%s", detail)
             return ""
 
+    def _get_address(self, index=0, ip_version="ipv4", session=None, timeout=60.0):
+        try:
+            return super()._get_address(index, ip_version, session, timeout)
+        except virt_vm.VMIPAddressMissingError:
+            mac = self.get_mac_address(index).lower()
+            ipaddr = utils_net.obtain_guest_ip_from_dhcp_leases(mac)
```

Comment:
I have another concern: we have this scenario: 
modifying interface type from network to bridge, but keep the mac address. Seems the dhcp-lease will not be deleted for a while, but the ip address of interface must change. There's a chance we're getting the 'old' ip address, right?